new-log-viewer/src/components/CentralContainer/ResizeHandle.css

```diff
@@ -0,0 +1,16 @@
+.resize-handle {
+    cursor: ew-resize;
+
+    z-index: var(--ylv-resize-handle-z-index);
+
+    width: var(--ylv-panel-resize-handle-width);
+    height: 100%;
+
+    /* stylelint-disable-next-line custom-property-pattern */
+    background-color: var(--joy-palette-neutral-outlinedBorder, #cdd7e1);
+}
+
+.resize-handle:hover {
+    /* stylelint-disable-next-line custom-property-pattern */
+    background-color: var(--joy-palette-primary-solidHoverBg, #0258a8);
+}
```

Comment:
Those CSS property names are from JoyUI which we cannot easily override. While it's possible to avoid referencing the properties in CSS style sheets by writing CSS-in-JS code to get the color platte, we avoid CSS-in-JS at the moment with the belief that it usually has worse performance than static stylesheets. We can revisit whether we wish to use CSS-in-JS once the Pigment engine is supported in JoyUI: https://mui.com/blog/introducing-pigment-css/